docs/reporting-flake.md

```diff
@@ -0,0 +1,32 @@
+---
+id: reporting-flake
+title: Reporting Flake
+---
+
+A powerful feature in the Happo review process is the ability to report a diff
+as flaky. To use this feature, press the "Report Flake" button next to any diff.
+
+Reporting a diff as flaky will instruct Happo to ignore that specific diff now
+and in the future. It should be used with a little bit of caution as it might
+hide issues that should be caught.
+
+Examples of where it _is okay_ to report a diff as flaky a diff as always
+ignored:
+
+- ✅ When there's a subtle diff that you can't attribute to the changes made.
+- ✅ When there are flaky rendering-related issues not caused by your code.
+
+Examples of where you _should avoid_ reporting a diff as flaky:
+
+- ❌ You've made a UI update causing diffs (simply
+  [accept the report](reviewing-diffs.md#acceptingrejecting-a-report) instead).
+- ❌ You're having spurious diffs where some images/fonts won't load
```

Comment:
Maybe change this to say "You believe the diffs are caused by a bug in Happo"? Or something similar which aren't related specifically to fonts/images (because these seem like proper flake).